packages/utils/src/ignore.js

```diff
@@ -0,0 +1,51 @@
+import path from 'path'
+import fs from 'fs-extra'
+import ignore from 'ignore'
+
+export class Ignore {
```

Comment:
If this util is only used for builder (build time) i suggest moving it to the builder package. To reduce utils install size.